src/Compilers/Core/Portable/DiagnosticAnalyzer/IAnalyzerPathResolver.cs

```diff
@@ -0,0 +1,52 @@
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
```

Comment:
file scoped namespace?

Reply:
Considered it but the rest of this area doesn't use them so followed the existing pattern for consistnency.

Reply:
wfm.  ide has been taking the approach of new code doing this.  but keeping consistency is totally valid.